manifests/config.pp

```diff
@@ -77,8 +78,32 @@
   $rdeck_base     = $framework_config['rdeck.base']
   $projects_dir   = $framework_config['framework.projects.dir']
   $properties_dir = $framework_config['framework.etc.dir']
+  $plugin_dir     = $framework_config['framework.libext.dir']
+
+  File[$rdeck_home] ~> File[$framework_config['framework.ssh.keypath']]
+
+  file { $rdeck_home:
+    ensure  => directory,
+  }
+
+  if $rundeck::sshkey_manage {
+    file { $framework_config['framework.ssh.keypath']:
+      mode    => '0600',
+    }
+  }
+
+  file { $rundeck::service_logs_dir:
+    ensure  => directory,
+  }
+
+  file { '/var/rundeck/':
```

Comment:
@juniorsysadmin do you think that setting file permissions here will trigger a reload of the service? Maybe this is why it was in the install class rather than the config class?